lib/transformer.dart

```diff
@@ -85,6 +100,47 @@ class InitializeTransformer extends Transformer {
       }
     });
   }
+
+  Future _replaceEntryWithBootstrap(Transform transform) {
+    // For now at least, _htmlEntryPoint, _entryPoint, and _newEntryPoint need
```

Comment:
Would it be enough if entrypoint/newentrypoint are not in lib? Then we can just use a relative path.